docs/docs/multi-dc.md

```diff
@@ -0,0 +1,176 @@
+multiple clusters in order to allow each of the clusters to coordinate and replicate data.  Periodically the seeds
+resource in the http API will be queried for each datacenter, when the seeds are able to be resolved they will be appended
+to the seed set when each server iss started.
+
+
+## Requirements
+
+1. Each individual cluster will need to be able to route to the IP addresses of each of the other clusters.
+2. All clusters need to have the same cluster name
+3. Datacenters need to be defined before starting the framework. New datacenters can not yet be added after a cluster has initialized.
+4. Node sizes and counts should be the same across all clusters.  The framework does not enforce or validate this, but should be configured equivalently on all clusters.
+
+## Providing Configuration
+
+### DC and Rack
+
+Configuration is provided via environment variables.
+```
+# Value to be used as the default DC name
+CASSANDRA_DEFAULT_DC=DC1
+
+# Value to be used as the default RACK name
+CASSANDRA_DEFAULT_RACK=RAC1
+
+```
+
+### Other instances of Cassandra on Mesos
+
+Cassandra on Mesos is informed of other instances of Cassandra on Mesos (running in another Mesos cluster) by
+environment variables of the format `CASSANDRA_EXTERNAL_DC_<dcName>=<cassandraOnMesosUrl>`.
+
+## Example on Mesos
+
+Imagine three Data Centers dc0, dc1, dc2 running three different Mesos clusters.
+
+The following configuration would be used for each instance of Cassandra on Mesos
+
+###### dc0
+```
+CASSANDRA_CLUSTER_NAME=multi-dc
+CASSANDRA_DEFAULT_DC=dc0
+CASSANDRA_EXTERNAL_DC_dc1=http://multi-dc.cassandra.marathon.mesos1:10001
+CASSANDRA_EXTERNAL_DC_dc2=http://multi-dc.cassandra.marathon.mesos2:10002
+```
+
+###### dc1
+```
+CASSANDRA_CLUSTER_NAME=multi-dc
+CASSANDRA_DEFAULT_DC=dc1
+CASSANDRA_EXTERNAL_DC_dc0=http://multi-dc.cassandra.marathon.mesos0:10000
+CASSANDRA_EXTERNAL_DC_dc2=http://multi-dc.cassandra.marathon.mesos2:10002
+```
+
+###### dc2
+```
+CASSANDRA_CLUSTER_NAME=multi-dc
+CASSANDRA_DEFAULT_DC=dc2
+CASSANDRA_EXTERNAL_DC_dc0=http://multi-dc.cassandra.marathon.mesos0:10000
+CASSANDRA_EXTERNAL_DC_dc1=http://multi-dc.cassandra.marathon.mesos1:10001
+```
+
+1. Ensure there are TCP routes allowing for traffic flow between each of the Mesos clusters.
+2. Start each cluster with the necessary configuration.
+3. Wait for each cluster to allocate it's seed nodes.
+4. After the seeds have been allocated a restart needs to be issued to all of the nodes that have already been started. Send a `POST` to the http API at `/cluster/rolling-restart/start` and the framework will restart each of the nodes so that new new seed set will take effect.
+5. Your clusters should now be able to replicate data between each other.
+
+## Example on DCOS
+
+Imagine three Data Centers dcos0, dcos1, dcos2 running three different DCOS Clusters.
+
+The following options files would be used for the install for each instance of Cassandra on DCOS
```

Comment:
`The following options files can be used to install each instance of Cassandra on DCOS.`